docs/opcodes/E4.mdx

```diff
@@ -0,0 +1,23 @@
+---
+fork: Prague
+group: Subroutine and Function
+---
+
+*Index 1 is top of the stack. See [PUSH](/#60).*
+
+## Notes
+
+Context from [CALLF](/#E3), The **RETF** instruction return value from a function or subroutine, which pop an execution state from return stack and set to corresponding values (current section index, program counter).
+
+Static jump instructions are prefered, dynamic jump instructions are disallowed.
```

Comment:
what should I put here?

Reply:
never mind, I think I just was confused when I first looked at this because I was reading the `RETF` and didn't connect it to the `CALLF`. Maybe just reword it into something like what I wrote below (and also do it for `CALLF`)

```
Within these functions/subroutines static jump instructions are preferred, dynamic jump instructions are disallowed.
```

Reply:
yeah, make sense!